BAZProjectC1/Components/CarouselMovies/movieCarouselCell.swift

```diff
@@ -0,0 +1,100 @@
+        movieRanking.textColor = UIColor.appColorWhitePrimary
+        movieRanking.textAlignment = .left
+        return movieRanking
+    }()
+    
+    private lazy var posterImage: UIImageView = {
+        let posterImage = UIImageView(frame: contentView.bounds)
+        posterImage.layer.cornerRadius = RounderBorderStyleForView.rounded.rawValue
+        posterImage.rounderCorners()
+        return posterImage
+    }()
+    
+    private lazy var movieRankingImage: UIImageView = {
+        let frame = CGRect(x: 0, y: 0, width: 16, height: 16)
+        let movieRankingImage = UIImageView(frame: frame)
+        movieRankingImage.translatesAutoresizingMaskIntoConstraints = false
+        movieRankingImage.image = UIImage(named: "startRanking")?.withTintColor(UIColor.appColorYellowPrimary)
+        return movieRankingImage
+    }()
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+    }
+    
+    func setupUI() {
+        contentView.addSubview(posterImage)
+        contentView.addSubview(backView)
+        contentView.addSubview(movieTitle)
+        contentView.addSubview(movieRankingImage)
+        contentView.addSubview(movieRanking)
+        movieRankingImage.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8).isActive = true
+        movieRankingImage.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16).isActive = true
+        movieRankingImage.widthAnchor.constraint(equalToConstant: 24).isActive = true
+        movieRankingImage.heightAnchor.constraint(equalToConstant: 24).isActive = true
+        movieRankingImage.heightAnchor.constraint(equalTo: movieRankingImage.widthAnchor, multiplier: contentView.frame.size.height / contentView.frame.size.width).isActive = true
+        movieRanking.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8).isActive = true
+        movieRanking.leftAnchor.constraint(equalTo: movieRankingImage.rightAnchor, constant: 8).isActive = true
+        movieTitle.widthAnchor.constraint(equalToConstant: contentView.frame.width).isActive = true
+        movieTitle.rightAnchor.constraint(equalTo: contentView.rightAnchor, constant: -16).isActive = true
+        movieTitle.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16).isActive = true
+        movieTitle.bottomAnchor.constraint(equalTo: movieRankingImage.topAnchor, constant: -8).isActive = true
+    }
+    
+    func configuration(dataInfo: MovieModel) {
+        posterImage.image = UIImage.imageFromColor(with: UIColor.appColorYellowPrimary.withAlphaComponent(0.1), size: posterImage.frame.size)
+        posterImage.loadImageFromUrl(urlString: "\(EndpointsList.imageResorce.description)\(dataInfo.poster_path ?? "")")
+        movieTitle.text = dataInfo.title
+        movieRanking.text = "\(Int(dataInfo.vote_average ?? 0))"
```

Comment:
Tal vez cierta lógica de aquí podría moverse al modelo `MovieModel`. Por ejemplo:

```
extension MovieModel {

   func posterImage(size: CGFloat): UIImage {
      UIImage.imageFromColor(with: UIColor.appColorYellowPrimary.withAlphaComponent(0.1), size: size)
   }

   var imageURLString: String {
      "\(EndpointsList.imageResorce.description)\(poster_path ?? "")"
   }

   var movieRanking: String {
      "\(Int(vote_average ?? 0))"
   }
}
```

Y después:
```suggestion
        posterImage.image = dataInfo.posterImage(size: posterImage.frame.size)
        posterImage.loadImageFromUrl(urlString: dataInfo.imageURLString)
        movieTitle.text = dataInfo.title
        movieRanking.text = dataInfo.movieRanking
```